README.md

```diff
@@ -39,9 +39,41 @@ View dependency in package.json:
 Search in tree:
 ![Search_In_Tree](resources/readme/gifs/search.gif)
 
-To filter scan results, click on the Filter ![Filter](resources/readme/filter.png) button.
+To filter scan results, click on the Filter ![Filter](resources/readme/filter.png) button:
 ![Filter](resources/readme/gifs/filter.gif)
 
+### Proxy
+
+To use HTTP/S proxy:
```

Comment:
To use HTTP/S proxy: --> If your JFrog Xray instance is behind an HTTP/S proxy, follow these steps to configure the proxy server.